src/components/overlay/overlay.js

```diff
@@ -146,7 +146,8 @@ export const BOverlay = /*#__PURE__*/ Vue.extend({
             'position-absolute': !this.noWrap || (this.noWrap && !this.fixed),
             'position-fixed': this.noWrap && this.fixed
           },
-          style: { ...positionCover, zIndex: this.zIndex || 10 }
+          style: { ...positionCover, zIndex: this.zIndex || 10 },
+          on: { click: evt => this.$emit('click', evt) }

```

Comment:
Maybe this should be `overlay-click`?

Reply:
thought about that too but I think it is fine like that.

Reply:
Vue 3 is going to make it difficult to use custom events names that are the same as native event names (they got rid of the `.native` modifier)

Reply:
as all future events will be added by `onClick="handler"` (i.e. as `$attrs`) where there is no option to bind a native listener (if users want to) to the root element :crying_cat_face: